elasticjob-cloud/elasticjob-cloud-executor/src/test/java/org/apache/shardingsphere/elasticjob/cloud/executor/prod/TaskExecutorTest.java

```diff
@@ -127,4 +130,20 @@ public void assertShutdown() {
     public void assertError() {
         taskExecutor.error(executorDriver, "");
     }
+    
+    @Test
+    @SneakyThrows
+    public void assertConstructor() {
+        TestSimpleJob testSimpleJob = new TestSimpleJob();
+        taskExecutor = new TaskExecutor(testSimpleJob);
+        Field fieldElasticJob = TaskExecutor.class.getDeclaredField("elasticJob");
+        fieldElasticJob.setAccessible(true);
+        Field fieldElasticJobType = TaskExecutor.class.getDeclaredField("elasticJobType");
+        fieldElasticJobType.setAccessible(true);
+        assertTrue(fieldElasticJob.get(taskExecutor) == testSimpleJob);
+        assertNull(fieldElasticJobType.get(taskExecutor));
+        taskExecutor = new TaskExecutor("simpleJob");
+        assertEquals("simpleJob", fieldElasticJobType.get(taskExecutor));
```

Comment:
Please use assertThat instead of assertEquals

Reply:
Ok, thanks for reviewing!